lib/index.js

```diff
@@ -39,6 +46,28 @@ export default function runContentfulImport (usageParams) {
     return Promise.reject(new Error('Either the `contentFile` or `content` option are required.'))
   }
 
+  const infoTable = new Table()
+
+  infoTable.push([{colSpan: 2, content: 'The following amount of entities will be imported:'}])
```

Comment:
I think it is more useful to see what is going to be imported before the actual import starts. So you get an overview what is going to happen.

We could display the same table at the end again, but it would display exactly the same since `pushToSpace` is not returning the entities that got successfully imported.